apis/training/v1alpha1/pytorchjob_types.go

```diff
@@ -46,6 +46,29 @@ type PyTorchJobSpec struct {
 	// CacheBackend is used to configure the cache engine for job
 	// +optional
 	CacheBackend *cachev1alpha1.CacheBackendSpec `json:"cacheBackend"`
+
+	// EnableElastic decides whether torch elastic is enabled for job.
+	// +optional
+	EnableElastic bool `json:"enableElastic"`
+
+	// ElasticPolicy is used to configure the torch elastic-based elastic scaling support for distributed training job.
+	// +optional
+	ElasticPolicy *ElasticPolicy `json:"elasticPolicy,omitempty"`
+}
+
+type ElasticPolicy struct {
+	// minReplicas is the lower limit for the number of replicas to which the training job
+	// can scale down.  It defaults to null.
+	MinReplicas *int32 `json:"minReplicas,omitempty"`
+
+	// upper limit for the number of pods that can be set by the autoscaler; cannot be smaller than MinReplicas, defaults to null.
+	MaxReplicas *int32 `json:"maxReplicas,omitempty"`
+
+	RDZVBackend  string `json:"rdzvBackend,omitempty"`
+	RdzvEndpoint string `json:"rdzvEndpoint"`
```

Comment:
is `rdzvEndpoint` required and must not `omitempty`?

Reply:
Yes, RDZVBackend and RdzvEndpoint are both required when EnableElastic is true.